payments-core/src/main/java/com/stripe/android/payments/core/authentication/DefaultPaymentAuthenticatorRegistry.kt

```diff
@@ -24,16 +24,16 @@ import javax.inject.Singleton
 import kotlin.coroutines.CoroutineContext
 
 private typealias AuthenticatorKey = Class<out StripeIntent.NextActionData>
-private typealias Authenticator = @JvmSuppressWildcards PaymentAuthenticator<StripeIntent>
+private typealias Authenticator = @JvmSuppressWildcards PaymentNextActionHandler<StripeIntent>
 
 /**
- * Default registry to provide look ups for [PaymentAuthenticator].
+ * Default registry to provide look ups for [PaymentNextActionHandler].
  * Should be only accessed through [DefaultPaymentAuthenticatorRegistry.createInstance].
  */
 @Singleton
 internal class DefaultPaymentAuthenticatorRegistry @Inject internal constructor(
-    private val noOpIntentAuthenticator: NoOpIntentAuthenticator,
-    private val sourceAuthenticator: SourceAuthenticator,
+    private val noOpIntentAuthenticator: NoOpIntentNextActionHandler,
+    private val sourceAuthenticator: SourceNextActionHandler,
```

Comment:
do you think updating variable names should also be in scope for this PR? e.g. I think the `Authenticator` typealias and then noOpIntentAuthenticator and sourceAuthenticator here could probably all be renamed

Reply:
Ok after looking at the rest of the PR, it does look like renaming variables would make the PR a lot bigger so I suspect that's why you didn't do it here. That makes sense -- are you planning to follow up and rename the variables?

Reply:
Yh, I plan to rename the variables in a follow up PR. The second Motivation section was meant to be a TODO section. I just updated it.

Reply:
gotchaa, thanks!